tests/acceptance/features/apiGraph/unassignRole.feature

```diff
@@ -17,11 +18,19 @@ Feature: unassign user role
     When user "Brian" uploads file with content "this step will assign the role to default" to "assign-to-default.txt" using the WebDAV API
     And user "Brian" should have the role "User" assigned
     Examples:
-      | user-role   |
-      | Admin       |
-      | Space Admin |
-      | User        |
-      | User Light  |
+      | user-role   | dav-path-version |
+      | Admin       | old              |
+      | Space Admin | old              |
+      | User        | old              |
+      | User Light  | old              |
+      | Admin       | new              |
+      | Space Admin | new              |
+      | User        | new              |
+      | User Light  | new              |
+      | Admin       | spaces           |
+      | Space Admin | spaces           |
+      | User        | spaces           |
+      | User Light  | spaces           |
```

Comment:
do graph api endpoint affected by dav-path-version?

Reply:
yeah, we can leave it as it is. no need to run with multiple dav paths (though we have one dav When step - the main actions is unassigning role)

Reply:
I added the `dav-path-version` because of the following steps:
```Scenario
When user "Brian" uploads file with content "this step will assign the role to default" to "assign-to-default.txt" using the WebDAV API
And user "Brian" should have the role "User" assigned
```

I will revert the changes back to original. :+1: